scripts/regression_test/extension_regression_test.yml

```diff
@@ -106,17 +106,20 @@ jobs:
   - template: ../../.azure-pipelines/templates/azdev_setup.yml
     parameters:
       CLIExtensionRepoPath: ./azure-cli-extensions
-  - bash: |
-      set -ev
-
-      source env/bin/activate
-      cd azure-cli-extensions
-
-      az login -u $(CLI_LIVE_TEST_ACCOUNT) -p "$(CLI_LIVE_TEST_PASSWORD)"
-      az account set -s 0b1f6471-1bf0-4dda-aec3-cb9272f09590
-
-      python ../scripts/ci/automation_full_test.py "12" "$(Instance_idx)" "latest" "" "True" "extension"
-    displayName: "Rerun tests"
+  - task: AzureCLI@1
```

Comment:
Any reason to use v1 instead of v2?

Reply:
totally fine to migrate to v2. It's just initially copied from somewhere else and v1 can already satisfy our requirements.